pkg/smartcontract/context/context.go

```diff
@@ -65,15 +65,23 @@ func (c *ParameterContext) GetWitness(h util.Uint160) (*transaction.Witness, err
 		}
 		emit.Bytes(bw.BinWriter, item.Parameters[i].Value.([]byte))
 	}
+	script := item.Script
+	if len(item.Parameters) == 0 {
```

Comment:
But what if the contract has parameters for `verify` method? I think that we may add `isDeployed` to `Item` struct.

Reply:
The problem is that `Item` struct is needed to be compatible with C# node.
I think we may check script for having `SYSCALL System.Contract.Call` or create an issue in main repo.

Reply:
Fixed it another way: we may set script to nil during item creation. This seems the best way to go, given that `script` meaning is "verification script to emit".